src/lib.rs

```diff
@@ -2029,7 +2044,7 @@ impl ByteOrder for BigEndian {
         unsafe {
             let bytes = *(&n.to_be() as *const u64 as *const [u8; 8]);
             copy_nonoverlapping(
-                bytes.as_ptr().offset((8 - nbytes) as isize),
+                bytes.as_ptr().add(8 - nbytes),
```

Comment:
I think `add` requires Rust 1.26+.

Reply:
Nice catch, thanks. I'll test with an old rustc just to make sure I didn't accidentally introduce something else.